ci/tasks/run-external-blobstore-validation-test/task.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -euxo pipefail
```

Comment:
Could you please remove the `set -x` option? We are trying to eliminate it from our CI tasks to reduce noise and avoid potential credential leaks.